parla/task_runtime.py

```diff
@@ -954,9 +891,11 @@ def stop(self):
     def run(self):
         pass
 
+
 class WorkerThreadException(RuntimeError):
     pass
 
+
```

Comment:
nice

Reply:
I sorta stopped reading closely below this line in this file, as it seems like very minor renaming and whitespace changes. LMK if I should look at anything more closely.